js/babel-plugin-sprockets-commoner-internal/index.js

```diff
@@ -198,6 +251,25 @@ module.exports = function (context) {
         }
         break;
       }
+    },
+    Identifier: function Identifier(path, state) {
```

Comment:
If so, there are some other globals for Node: https://github.com/sindresorhus/globals/blob/master/globals.json#L840-L860. Many of these are shared between Node and the browser, others are not super relevant to us, so maybe not a big deal. I could definitely see `__dirname` coming into play, maybe `setImmediate` too.